src/SFA.DAS.Commitments.Application/Commands/CohortApproval/ProiderApproveCohort/ProviderApproveCohortCommandHandler.cs

```diff
@@ -7,24 +7,21 @@
 using SFA.DAS.Commitments.Application.Interfaces.ApprenticeshipEvents;
 using SFA.DAS.Commitments.Application.Rules;
 using SFA.DAS.Commitments.Application.Services;
-using SFA.DAS.Commitments.Domain;
 using SFA.DAS.Commitments.Domain.Data;
 using SFA.DAS.Commitments.Domain.Entities;
 using SFA.DAS.Commitments.Domain.Interfaces;
-using SFA.DAS.Commitments.Events;
-using SFA.DAS.Messaging.Interfaces;
+using SFA.DAS.CommitmentsV2.Types;
+using EditStatus = SFA.DAS.Commitments.Domain.Entities.EditStatus;
 
 namespace SFA.DAS.Commitments.Application.Commands.CohortApproval.ProiderApproveCohort
```

Comment:
Not sure if this was part of this story but there appears to be a typo in in namespace